VideoWeb/VideoWeb/ClientApp/src/app/shared/force-play-video.directive.ts

```diff
@@ -1,14 +1,14 @@
-import { Directive, ElementRef, OnInit, Renderer2, RendererFactory2 } from '@angular/core';
+import { Directive, ElementRef, OnDestroy, OnInit, Renderer2, RendererFactory2 } from '@angular/core';
 import { Logger } from '../services/logging/logger-base';
 
 @Directive({
     selector: '[appForcePlayVideo]'
 })
-export class ForcePlayVideoDirective implements OnInit {
+export class ForcePlayVideoDirective implements OnInit, OnDestroy {
     private readonly loggerPrefix = '[ForcePlayVideoDirective] -';
     private renderer: Renderer2;
-    private unsubscribeFromMouseDownCallback: () => void;
-    private unsubscribeFromTouchStartCallback: () => void;
+    private unsubscribeFromMouseDownCallback: () => void | null = null;
+    private unsubscribeFromTouchStartCallback: () => void | null = null;
```

Comment:
What is the purpose of null = null here?

Reply:
The first null is part of the type definition. The second null is the default value.

Reply:
That seems very obvious now :D Thanks